packages/ember-auto-import/ts/broccoli-append.ts

```diff
@@ -200,7 +209,7 @@ interface PassthroughEntry extends WalkSyncEntry {
 
 type AugmentedWalkSyncEntry = WalkSyncEntry | PassthroughEntry;
 
-function findByPrefix(path: string, map: Map<string, string>) {
+function findByPrefix(path: string, map: Map<string, any>) {
```

Comment:
I think we can replace `any` here with `Map<string, string>`?

Reply:
It can't just be `Map<string, string>` because we call it with both `this.passthrough` and `this.mappings`, and this PR changes `this.mappings` to `Map<string, Map<string, string>>` (but leaves `this.passthrough` as `Map<string, string>`). So maybe we could do `Map<string, string | Map<string, string>>` but I thought since the function itself is agnostic as to what the values of the map are, it made sense to leave it as `any`. But I'm happy to change it to an `|` if you prefer.

Reply:
Ah, in that case it should be:

```ts
function findByPrefix<T>(path: string, map: Map<string, T>) {
```

Reply:
Ah, TIL! I'm still kinda new to typescript also. I just pushed a commit (didn't squash/rebase or anything -- figured you could just do that when merging).